erpnext/manufacturing/doctype/job_card/job_card.py

```diff
@@ -80,6 +80,12 @@ def get_required_items(self):
 					'required_qty': (d.required_qty * flt(self.for_quantity)) / doc.qty
 				})
 
+	def get_scrap_items(self):
+		self.scrap_items = []
```

Comment:
Do you need this line? I think the `self.set` will reset it anyway

Reply:
yeah self.set will reset it